### PR TITLE
toot: init at 0.8.0 (including pythonPackages.beautifulsoup4 update)

### PR DESCRIPTION
**Pull request by matthiasbeyer**
###### Motivation for this change

Mastodon CLI interface.

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/toot/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, pythonPackages, w3m, file, less }:
```
Comment (FRidh) [inline pkgs/applications/misc/toot/default.nix]: what are `file` and `less` for?

File: pkgs/applications/misc/toot/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  };
+
+  propagatedBuildInputs = with pythonPackages;
+    [ requests2 beautifulsoup4 future ];
+
+  meta = with stdenv.lib; {
+    description = "Mastodon CLI interface";
+    homepage    = "https://github.com/ihabunek/toot";
+    license     = licenses.mit;
+    platforms   = platforms.unix;
```
Comment (FRidh) [inline pkgs/applications/misc/toot/default.nix]: no need to set platforms as it is already done correctly by `buildPythonApplication`

**Review by FRidh — CHANGES_REQUESTED** [review]
Two minor changes are needed.
Furthermore, tests require pytest.

**Comment by matthiasbeyer** [comment]
Fixed.

**Pull request merged**